pkg/proxy/ipvs/README.md

```diff
@@ -334,31 +334,8 @@ export KUBE_PROXY_MODE=ipvs
 
 ### Cluster Created by Kubeadm
 
-If you are using kubeadm with a [configuration file](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file), you have to add `mode: ipvs` and also add `SupportIPVSProxyMode: true` below the `kubeProxy` field as part of the kubeadm configuration.
-
-```yaml
-...
-kubeProxy:
-  config:
-    featureGates:
-      SupportIPVSProxyMode: true
-    mode: ipvs
-...
-```
 Note that in Kubernetes 1.11 and later, `SupportIPVSProxyMode` is set to `true` by default.
```

Comment:
This can be removed, as the SupportIPVSProxyMode is being completely removed from the code